web_src/js/utils/match.ts

```diff
@@ -41,3 +44,13 @@ export function matchMention(queryText: string): string[] {
 
   return sortAndReduce(results);
 }
+
+export async function matchIssue(owner: string, repo: string, issueIndexStr: string, query: string): Promise<Issue[]> {
+  const res = await GET(`${window.location.origin}${window.config.appSubUrl}/${owner}/${repo}/-/issues/suggestions?q=${encodeURIComponent(query)}`);
```

Comment:
That's not what other code does .....

![image](https://github.com/user-attachments/assets/3559c17b-ec3b-481c-a25e-12abc848fcb6)